api/src/test/java/com/ford/labs/retroquest/api/BoardApiTest.java

```diff
@@ -163,6 +164,40 @@ public void createBoard_WithUnauthorizedUser_Returns403() throws Exception {
             .andExpect(status().isForbidden());
     }
 
+    @Test
+    public void deleteBoard_shouldDeleteSingleBoardSuccessfully() throws Exception {
+        Board boardToDelete =  boardRepository.save(Board.builder()
+                .dateCreated(LocalDate.of(2018, 3, 3))
+                .teamId(teamId)
+                .thoughts(List.of())
+                .build());
+
+        assertThat(boardRepository.count()).isEqualTo(1);
+
+        mockMvc.perform(delete(String.format("/api/team/%s/boards/%s", teamId, boardToDelete.getId()))
+                        .contentType(APPLICATION_JSON)
+                        .header("Authorization", getBearerAuthToken()))
+                .andExpect(status().isOk());
+
+        assertThat(boardRepository.count()).isEqualTo(0);
+    }
+
+    @Test
+    public void deleteBoard_withUnauthorizedUser_shouldReturn403() throws Exception {
+        Board boardToDelete =  boardRepository.save(Board.builder()
+                .dateCreated(LocalDate.of(2018, 3, 3))
+                .teamId(teamId)
+                .thoughts(List.of())
+                .build());
+
+        mockMvc.perform(delete(String.format("/api/team/%s/boards/%s", teamId, boardToDelete.getId()))
+                        .contentType(APPLICATION_JSON)
+                        .header("Authorization", "Bearer " + jwtBuilder.buildJwt("unauthorized")))
+                .andExpect(status().isForbidden());
+
+        assertThat(boardRepository.count()).isEqualTo(1);
+    }
+
```

Comment:
* What's the expected behavior of this endpoint if we ask it to delete a board it can't find, and 
* should that be tested?

Reply:
Somehow I missed these comments before I merged the PR in. Is there anything we'd want it to do? If the board doesn't exist, its already deleted, so, nothing?